common/static/common/js/components/collections/paging_collection.js

```diff
@@ -92,6 +102,26 @@
                 );
             },
 
+
+            /**
+             * Refreshes the collection if it has been marked as stale.
+             * @param force If true, it will always refresh.
+             * @returns {promise} Returns a promise representing the refresh.
+             */
+            refresh: function(force) {
```

Comment:
As far as I can tell we're not using the `force` option anywhere. Do we need to keep it?